client/pages/DeckStat.jsx

```diff
@@ -0,0 +1,186 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Col, Row } from 'react-bootstrap';
+import { useTranslation } from 'react-i18next';
+import Panel from '../Components/Site/Panel';
+import CardBack from '../Components/Decks/CardBack';
+import { loadStats } from '../redux/actions';
+import { clearApiStatus } from '../redux/actions';
+
+import { BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend, Scatter,Radar, RadarChart, PolarGrid, PolarAngleAxis, PolarRadiusAxis, } from 'recharts';
+// const data = [
+//     {
+//         level: 'Beginner',
+//         win: 40,
+//         loss: 2,
+//         ratio: 89
+//     },
+//     {
+//         level: 'Casual',
+//         win: 40,
+//         loss: 40,
+//         ratio: 50
+//     },
+//     {
+//         level: 'Competitive',
+//         win: 20,
+//         loss: 80,
+//         ratio: 20
+//     }
+// ];
+
+// const renderCustomizedLabel = (props) => {
+//     const { x, y, width, value } = props;
+//     const radius = 10;
+//     return (
+//       <g>
+//         <circle cx={x + width / 2} cy={y - radius} r={radius} fill="#8884d8" />
+//         <text x={x + width / 2} y={y - radius} fill="#fff" textAnchor="middle" dominantBaseline="middle">
+//           {value.split(' ')[1]}
+//         </text>
+//       </g>
+//     );
+//   };
+const DecksComponent = () => {
+    const { user } = useSelector((state) => ({
+        user: state.account.user
+    }));
+    const { deck } = useSelector((state) => ({
+        deck: state.cards.selectedDeck
+    }));
+    let { stats } = useSelector((state) => ({
+        stats: state.stats
+    }));
+    const dispatch = useDispatch();
+    const { t } = useTranslation();
+
+    useEffect(() => {
+            // if (user != null && deck != null) {
+            dispatch(
+                loadStats({
```

Comment:
And 2nd probleme, i dont know how the load the result of this api call into the variable stats

Reply:
You don't. It's the useSelector line above